app/controllers/api/v1/user_apartments_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::UserApartmentsController < ApplicationController
+  def index
+    @user_favourite = UserApartment.where(user_id: 2)
+    @user_apartments = Apartment.where(id: @user_favourite.map(&:apartment_id))
```

Comment:
- [ ] Try to use the the `includes` method every time you try to get data from two tables to improve the performance